larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.cc

```diff
@@ -0,0 +1,352 @@
+/**
+ *  @file   larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.cc
+ *
+ *  @brief  Implementation of the deep learning track shower id algorithm.
+ *
+ *  $Log: $
+ */
+
+#include "Pandora/AlgorithmHeaders.h"
+
+#include <torch/script.h>
+#include <torch/torch.h>
+
+#include "larpandoradlcontent/LArDeepLearning/DeepLearningTrackShowerIdAlgorithm.h"
+#include "larpandoradlcontent/LArHelpers/LArDLHelper.h"
+#include "larpandoracontent/LArHelpers/LArFileHelper.h"
```

Comment:
Might typically have an empty line between the larpandoradlcontent and larpandoracontent includes?